src/lib/minions/data/killableMonsters/bosses/misc.ts

```diff
@@ -163,6 +163,20 @@ const killableBosses: KillableMonster[] = [
 		levelRequirements: {
 			prayer: 43
 		}
+	},
+	{
+		id: Monsters.Obor.id,
+		name: Monsters.Obor.name,
+		aliases: Monsters.Obor.aliases,
+		timeToFinish: Time.Second * 40,
```

Comment:
this seems pretty fast i would prob expect like a min to a min and a half, hes a f2p boss so kidna balance it around f2p gear not super overpowered p2p gear